ctms/crud.py

```diff
@@ -96,12 +96,14 @@ def _contact_base_query(db):
         .options(joinedload(Email.amo))
         .options(joinedload(Email.fxa))
         .options(joinedload(Email.mofo))
-        .options(selectinload("newsletters"))
-        .options(selectinload("waitlists"))
-        .options(joinedload(Email.stripe_customer))
-        .options(selectinload("stripe_customer.subscriptions"))
-        .options(selectinload("stripe_customer.subscriptions.subscription_items"))
-        .options(selectinload("stripe_customer.subscriptions.subscription_items.price"))
+        .options(selectinload(Email.newsletters))
+        .options(selectinload(Email.waitlists))
+        .options(
+            joinedload(Email.stripe_customer)
+            .subqueryload(StripeCustomer.subscriptions)
+            .subqueryload(StripeSubscription.subscription_items)
+            .subqueryload(StripeSubscriptionItem.price)
+        )
```

Comment:
calling `get_email()` from a Python shell with the `main` branch leads to the same SQL on the Postgres side:
```sql
SELECT
        emails.email_id AS emails_email_id
        ,emails.primary_email AS emails_primary_email
        ,emails.basket_token AS emails_basket_token
        ,emails.sfdc_id AS emails_sfdc_id
        ,emails.first_name AS emails_first_name
        ,emails.last_name AS emails_last_name
        ,emails.mailing_country AS emails_mailing_country
        ,emails.email_format AS emails_email_format
        ,emails.email_lang AS emails_email_lang
        ,emails.double_opt_in AS emails_double_opt_in
        ,emails.has_opted_out_of_email AS emails_has_opted_out_of_email
        ,emails.unsubscribe_reason AS emails_unsubscribe_reason
        ,emails.create_timestamp AS emails_create_timestamp
        ,emails.update_timestamp AS emails_update_timestamp
        ,fxa_1.id AS fxa_1_id
        ,fxa_1.fxa_id AS fxa_1_fxa_id
        ,fxa_1.email_id AS fxa_1_email_id
        ,fxa_1.primary_email AS fxa_1_primary_email
        ,fxa_1.created_date AS fxa_1_created_date
        ,fxa_1.lang AS fxa_1_lang
        ,fxa_1.first_service AS fxa_1_first_service
        ,fxa_1.account_deleted AS fxa_1_account_deleted
        ,fxa_1.create_timestamp AS fxa_1_create_timestamp
        ,fxa_1.update_timestamp AS fxa_1_update_timestamp
        ,amo_1.id AS amo_1_id
        ,amo_1.email_id AS amo_1_email_id
        ,amo_1.add_on_ids AS amo_1_add_on_ids
        ,amo_1.display_name AS amo_1_display_name
        ,amo_1.email_opt_in AS amo_1_email_opt_in
        ,amo_1.language AS amo_1_language
        ,amo_1.last_login AS amo_1_last_login
        ,amo_1.location AS amo_1_location
        ,amo_1.profile_url AS amo_1_profile_url
        ,amo_1. "user" AS amo_1_user
        ,amo_1.user_id AS amo_1_user_id
        ,amo_1.username AS amo_1_username
        ,amo_1.create_timestamp AS amo_1_create_timestamp
        ,amo_1.update_timestamp AS amo_1_update_timestamp
        ,mofo_1.id AS mofo_1_id
        ,mofo_1.email_id AS mofo_1_email_id
        ,mofo_1.mofo_email_id AS mofo_1_mofo_email_id
        ,mofo_1.mofo_contact_id AS mofo_1_mofo_contact_id
        ,mofo_1.mofo_relevant AS mofo_1_mofo_relevant
        ,stripe_customer_1.stripe_id AS stripe_customer_1_stripe_id
        ,stripe_customer_1.fxa_id AS stripe_customer_1_fxa_id
        ,stripe_customer_1.default_source_id AS stripe_customer_1_default_source_id
        ,stripe_customer_1.invoice_settings_default_payment_method_id AS stripe_customer_1_invoice_settings_default_payment_method_1
        ,stripe_customer_1.stripe_created AS stripe_customer_1_stripe_created
        ,stripe_customer_1.deleted AS stripe_customer_1_deleted
        ,stripe_customer_1.create_timestamp AS stripe_customer_1_create_timestamp
        ,stripe_customer_1.update_timestamp AS stripe_customer_1_update_timestamp
    FROM
        emails LEFT OUTER JOIN fxa AS fxa_1
            ON emails.email_id = fxa_1.email_id LEFT OUTER JOIN amo AS amo_1
            ON emails.email_id = amo_1.email_id LEFT OUTER JOIN mofo AS mofo_1
            ON emails.email_id = mofo_1.email_id LEFT OUTER JOIN (
            fxa AS fxa_2 JOIN stripe_customer AS stripe_customer_2
                ON fxa_2.fxa_id = stripe_customer_2.fxa_id JOIN stripe_customer AS stripe_customer_1
                ON fxa_2.fxa_id = stripe_customer_1.fxa_id
        )
            ON emails.email_id = fxa_2.email_id
    WHERE
        emails.email_id = '62d8d3c6-95f3-4ed6-b176-7f69acff22f6' : : uuid
```